src/main/scala/advancepassengerinfo/health/HealthRoute.scala

```diff
@@ -0,0 +1,23 @@
+package advancepassengerinfo.health
+
+import akka.http.scaladsl.model.StatusCodes
+import scala.concurrent.ExecutionContext
+import akka.http.scaladsl.server.Directives._
+import akka.http.scaladsl.server.Route
+
+object HealthRoute {
+
+  def checkHealth(processState: ProcessState)(implicit ec: ExecutionContext): Route =
+    get {
+      if (processState.isLatest)
```

Comment:
Very unclear what `processState.isLatest` means. Is latest what?
You need to think about what `ProcessState` represents, and name accordingly. This function should probably take a `FiniteDuration` and be called something like `hasCheckedSince`